includes/API/WriteStoreSignedTxHandler.php

```diff
@@ -14,10 +14,13 @@
 use Wikimedia\ParamValidator\ParamValidator;
 use WikiPage;
 
+// Typing
+use User;
```

Comment:
Is it better to explicitly import `User`, or just use `\User` in the type annotation?